--- ucblogo-6.0/makefile.in.orig	2013-05-15 19:59:50.770414901 +0200
+++ ucblogo-6.0/makefile.in	2013-05-16 17:07:42.770555795 +0200
@@ -6,7 +6,7 @@
 LIBS  = @X_PRE_LIBS@ @X_LIBS@ @LIBS@ @X_EXTRA_LIBS@
 prefix = @prefix@
 BINDIR        = $(prefix)/bin
-LIBLOC        = $(prefix)/lib/logo
+LIBLOC        = $(prefix)/share/logo
 LINKER = @LINKER@
 
 # LIBLOC      = `pwd`
@@ -71,11 +71,11 @@
 	cd docs; $(MAKE) ship
 
-install: all
-	for d in $(BINDIR) $(LIBLOC) $(LIBLOC)/logolib $(LIBLOC)/helpfiles $(LIBLOC)/csls; do [ -d $$d ] || mkdir -p $$d || exit 1; done
-	cp logo $(BINDIR)/.
-	cp -f logolib/* $(LIBLOC)/logolib/.
-	cp -f helpfiles/* $(LIBLOC)/helpfiles/.
-	cp -f csls/* $(LIBLOC)/csls/.
+install: logo
+	for d in $(DESTDIR)$(BINDIR) $(DESTDIR)$(LIBLOC) $(DESTDIR)$(LIBLOC)/logolib $(DESTDIR)$(LIBLOC)/helpfiles $(DESTDIR)$(LIBLOC)/csls; do [ -d $$d ] || mkdir -p $$d || exit 1; done
+	cp logo $(DESTDIR)$(BINDIR)/.
+	cp -f logolib/* $(DESTDIR)$(LIBLOC)/logolib/.
+	cp -f helpfiles/* $(DESTDIR)$(LIBLOC)/helpfiles/.
+	cp -f csls/* $(DESTDIR)$(LIBLOC)/csls/.
 #	(cd emacs; prefix=$(prefix) LIBLOC=$(LIBLOC) BINDIR=$(BINDIR) $(MAKE) install)
 	(cd docs; prefix=$(prefix) LIBLOC=$(LIBLOC) BINDIR=$(BINDIR) $(MAKE) install)
 #	prefix=$(prefix); LIBLOC=$(LIBLOC); BINDIR=$(BINDIR); export prefix LIBLOC BINDIR; cd emacs; $(MAKE) install
--- ucblogo-6.0/docs/makefile.orig	2008-08-19 21:56:59.000000000 +0200
+++ ucblogo-6.0/docs/makefile	2013-05-16 17:08:36.163886889 +0200
@@ -1,5 +1,5 @@
 BUILDIR	      = `pwd`
-INFODIR	      = $(prefix)/info
+INFODIR	      = $(prefix)/share/info
 DOCSDIR	      = $(LIBLOC)/docs
 HTMLDIR	      = $(DOCSDIR)/html
 
@@ -31,10 +31,10 @@
 	-rm -f *.{aux,cp,cps,dvi,fn,ky,log,pg,toc,tp,vr}
 
 install: all
-	for d in $(INFODIR) $(DOCSDIR) $(HTMLDIR); do [ -d $$d ] || mkdir -p $$d || exit 1; done
-	-cp -f *.info* $(INFODIR)/.
-	-cp -f html/*.html $(HTMLDIR)/.
-	-cp -f usermanual.ps $(DOCSDIR)/.
-	-cp -f usermanual.pdf $(DOCSDIR)/.
-	-cp -f usermanual.texi $(DOCSDIR)/.
-	-cp -f ../usermanual $(DOCSDIR)/.
+	for d in $(INFODIR) $(DOCSDIR) $(HTMLDIR); do [ -d $(DESTDIR)$$d ] || mkdir -p $(DESTDIR)$$d || exit 1; done
+	-cp -f *.info* $(DESTDIR)$(INFODIR)/.
+	-cp -f html/*.html $(DESTDIR)$(HTMLDIR)/.
+	-cp -f usermanual.ps $(DESTDIR)$(DOCSDIR)/.
+	-cp -f usermanual.pdf $(DESTDIR)$(DOCSDIR)/.
+	-cp -f usermanual.texi $(DESTDIR)$(DOCSDIR)/.
+	-cp -f ../usermanual $(DESTDIR)$(DOCSDIR)/.
