--- ucblogo-6.0/wxTerminal.cpp.orig	2014-01-18 16:49:02.000000000 +0100
+++ ucblogo-6.0/wxTerminal.cpp	2014-01-18 17:20:01.188770053 +0100
@@ -506,7 +507,7 @@
 #else
 			      wxStandardPaths::Get().GetDocumentsDir() :
 #endif
-			      *wxEmptyString),
+			      ""),
 			    wxEmptyString,
 			    _T("Logo workspaces(*.lg)|*.lg|All files(*)|*"),
 //			    "*",
@@ -542,7 +543,7 @@
 #else
 	    wxStandardPaths::Get().GetDocumentsDir() :
 #endif
-			  *wxEmptyString),
+			  ""),
 	 wxEmptyString,
 	 _T("Logo workspaces(*.lg)|*.lg|All files(*)|*"),
 //	 "*",
